Makefile

```diff
@@ -75,7 +75,8 @@ pkg/generated: $(API_TYPES)
 	touch $@
 
 crd-schema-gen:
-	$(GO) run ./vendor/github.com/openshift/crd-schema-gen/cmd/crd-schema-gen/ --apis-dir pkg/apis --manifests-dir manifests/
```

Comment:
don't do this. Get the binary release. Never vendor with another set of dependencies which are untested!

Reply:
Thanks for the comment @sttts ,  Cannot fetch the binary release in an environment that has no Internet access.  Are you suggesting I fetch the binary release and generate the CRs only in an environment with Internet access or is it ok to vendor in a specific release of "github.com/openshift/crd-schema-gen"?